packages/app-desktop/gui/NoteEditor/utils/contextMenu.ts

```diff
@@ -112,7 +113,12 @@ export function menuItems(): ContextMenuItems {
 		paste: {
 			label: _('Paste'),
 			onAction: async (options: ContextMenuOptions) => {
-				const content = clipboard.readHTML() ? clipboard.readHTML() : clipboard.readText();
+				let content = clipboard.readHTML() ? clipboard.readHTML() : clipboard.readText();
+
+				if (clipboard.readHTML()) {
+					content = await processPastedHtml(clipboard.readHTML());
+				}
```

Comment:
Please store the result of readHTML in a variable so as not to read it twice.

Reply:
done